### PR TITLE
Update outdated MAX links

### PR DESCRIPTION
**Pull request by ckadner**
Update outdated MAX links

/cc @ptitzler @Tomcli

### CODE REVIEW

File: model-samples/max-ocr/max-ocr.yaml [hunk 1]
```diff
@@ -2,16 +2,15 @@
...
-description: IBM Model Asset eXchange(MAX) model that identifies text in an image.
+description: "Identify text in an image"
 framework:
   name: "N/A"
   version: "N/A"
-
 license: "Apache 2.0"
 domain: "Optical Character Recognition"
-website: "https://developer.ibm.com/exchanges/models/all/max-ocr/"
+website: https://developer.ibm.com/exchanges/models/all/max-ocr/
```
Comment (ptitzler) [inline model-samples/max-ocr/max-ocr.yaml:13]: yields 404. Use https://github.com/IBM/MAX-OCR

File: model-samples/max-question-answering/max-question-answering.yaml [hunk 2]
```diff
@@ -2,16 +2,15 @@
...
-description: IBM Model Asset eXchange(MAX) model that answers questions on a given corpus of text
+description: "Answers questions on a given corpus of text"
 framework:
   name: "tensorflow"
   version: "1.15.4"
-
 license: "Apache 2.0"
 domain: "Question and Answer"
-website: "https://developer.ibm.com/exchanges/models/all/max-question-answering/"
+website: https://developer.ibm.com/exchanges/models/all/max-question-answering/
```
Comment (ptitzler) [inline model-samples/max-question-answering/max-question-answering.yaml:13]: yields 404. use https://github.com/IBM/MAX-Question-Answering

File: model-samples/max-text-sentiment-classifier/max-text-sentiment-classifier.yaml [hunk 3]
```diff
@@ -2,16 +2,15 @@
...
-description: IBM Model Asset eXchange(MAX) model that detects the sentiment captured in short pieces of text
+description: "Detect the sentiment captured in short snippets of text"
 framework:
   name: tensorflow
   version: "1.15.4"
-
 license: "Apache 2.0"
 domain: "Sentiment Analysis"
-website: "https://developer.ibm.com/technologies/artificial-intelligence/models/max-text-sentiment-classifier/"
+website: https://developer.ibm.com/technologies/artificial-intelligence/models/max-text-sentiment-classifier/
```
Comment (ptitzler) [inline model-samples/max-text-sentiment-classifier/max-text-sentiment-classifier.yaml:13]: https://github.com/IBM/MAX-Text-Sentiment-Classifier

File: model-samples/max-toxic-comment-classifier/max-toxic-comment-classifier.yaml [hunk 4]
```diff
@@ -2,16 +2,15 @@
...
-description: IBM Model Asset eXchange(MAX) model that detects 6 types of toxicity in user comments
+description: "Detect 6 types of toxicity in user comments"
 framework:
   name: pytorch
   version: "1.0.1"
-
 license: "Apache 2.0"
 domain: "Text Classification"
-website: "https://developer.ibm.com/exchanges/models/all/max-toxic-comment-classifier/"
+website: https://developer.ibm.com/exchanges/models/all/max-toxic-comment-classifier/
```
Comment (ptitzler) [inline model-samples/max-toxic-comment-classifier/max-toxic-comment-classifier.yaml:13]: https://github.com/IBM/MAX-Toxic-Comment-Classifier

File: model-samples/max-weather-forecaster/max-weather-forecaster.yaml [hunk 5]
```diff
@@ -2,12 +2,11 @@
...
+name: "Weather Forecaster"
 model_identifier: max-weather-forecaster
-description: IBM Model Asset eXchange(MAX) model that predicts hourly weather features given historical data for a specific location
+description: "Predict hourly weather features given historical data for a specific location"
 framework:
   name: keras
-
 license: "Apache 2.0"
 domain: "Time Series Prediction"
 website: "https://developer.ibm.com/exchanges/models/all/max-weather-forecaster"
```
Comment (ptitzler) [inline model-samples/max-weather-forecaster/max-weather-forecaster.yaml:12]: https://github.com/IBM/MAX-Weather-Forecaster

File: model-samples/max-object-detector/max-object-detector.yaml [hunk 6]
```diff
@@ -2,16 +2,15 @@
...
-description: IBM Model Asset eXchange(MAX) model that localizes and identifies multiple objects in a single image
+description: "Localize and identify multiple objects in a single image"
 framework:
   name: tensorflow
   version: "1.5.0"
-
 license: "Apache 2.0"
 domain: "Object detection in images"
-website: "https://developer.ibm.com/exchanges/models/all/max-object-detector"
+website: https://developer.ibm.com/exchanges/models/all/max-object-detector
```
Comment (ptitzler) [inline model-samples/max-object-detector/max-object-detector.yaml:13]: https://github.com/IBM/MAX-Object-Detector

File: model-samples/max-named-entity-tagger/max-named-entity-tagger.yaml [hunk 7]
```diff
@@ -2,16 +2,15 @@
...
-description: IBM Model Asset eXchange(MAX) model that locates and tags named entities in text.
+description: "Locate and tag named entities in a text"
 framework:
   name: tensorflow
   version: "1.15.4"
-
 license: "Apache 2.0"
 domain: "Natural Language Processing"
-website: "https://developer.ibm.com/exchanges/models/all/max-named-entity-tagger/"
+website: https://developer.ibm.com/exchanges/models/all/max-named-entity-tagger/
```
Comment (ptitzler) [inline model-samples/max-named-entity-tagger/max-named-entity-tagger.yaml:13]: https://github.com/IBM/MAX-Named-Entity-Tagger

File: model-samples/max-image-resolution-enhancer/max-image-resolution-enhancer.yaml [hunk 8]
```diff
@@ -2,16 +2,15 @@
...
-description: IBM Model Asset eXchange(MAX) model that upscales an image by a factor of 4, while generating photo-realistic details.
+description: "Upscale an image by a factor of 4 while generating photo-realistic details"
 framework:
   name: "tensorflow"
   version: "1.15.4"
-
 license: "Apache 2.0"
 domain: "Super-Resolution"
-website: "https://developer.ibm.com/exchanges/models/all/max-image-resolution-enhancer/"
+website: https://developer.ibm.com/exchanges/models/all/max-image-resolution-enhancer/
```
Comment (ptitzler) [inline model-samples/max-image-resolution-enhancer/max-image-resolution-enhancer.yaml:13]: https://github.com/IBM/MAX-Image-Resolution-Enhancer

File: model-samples/max-human-pose-estimator/max-human-pose-estimator.yaml [hunk 9]
```diff
@@ -2,15 +2,15 @@
...
 model_identifier: max-human-pose-estimator
-description: IBM Model Asset eXchange(MAX) model that detects humans in an image and estimate the pose for each person.
+description: "Detect humans in an image and estimate the pose for each person"
 framework:
   name: tensorflow
   version: "1.15.4"
 license: "Apache 2.0"
 domain: "Human Pose Estimation"
-website: "https://developer.ibm.com/exchanges/models/all/max-human-pose-estimator/"
+website: https://developer.ibm.com/exchanges/models/all/max-human-pose-estimator/
```
Comment (ptitzler) [inline model-samples/max-human-pose-estimator/max-human-pose-estimator.yaml:13]: https://github.com/IBM/MAX-Human-Pose-Estimator

**Comment by ckadner** [comment]
Thank you @ptitzler -- I had to fix the check-doc-links script to properly catch all defunct URLs

**Review by Tomcli — APPROVED** [review]

**Pull request merged**